attester-chrome.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ "$TRAVIS" = "true" ]; then
+    exit
+fi
+
+(node node_modules/attester/bin/attester.js test/attester-phantom-complementary.yml --env package.json &) && selenium-java-robot --browser 'chrome'
```

Comment:
Maybe the full command line could be stored directly in the `package.json` file (especially if the check of the `$TRAVIS` variable is no longer required if we also run those tests on travis).